infra/azure/terraform/README.md

```diff
@@ -0,0 +1,21 @@
+# infra/azure
+Terraform scripts and documentation for capz infrastructure that the Kubernetes community runs on Azure.
+The capz folder structure is formatted shown below:
+main.tf
+├── resource_group
+├── identities
+├── key_vault
+├── container_registry
+├── storage_account
+├── role_assignments
+├── variables
+
+# Prerequiste 
+The az-cli-prow service principal is required to be created by a tenant admin. Also to add a federated credentials for the prow ask a tenant admin or the SP admin. iam-config\param.json can be used for creating the fededrated credentials. 
+TODO: Add EKS federated creds 

```

Comment:
it will look like: 

```
{
    "name": "prow-eks-identity-credential",
    "issuer": "https://oidc.eks.us-east-2.amazonaws.com/id/F8B73554FE6FBAF9B19569183FB39762",
    "subject": "system:serviceaccount:test-pods:default",
    "audiences": [
        "api://AzureADTokenExchange"
    ]
}
```